pom.xml

```diff
@@ -119,7 +119,13 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
-	</dependencies>
+        <dependency>
```

Comment:
I thought we already had junit included. Is there a feature in version 5.6.2 that you needed to use?

Reply:
Good question..I'm not sure why or when I added that. Removed.